docs/src/docs/asciidoc/user/writers.adoc

```diff
@@ -98,3 +96,29 @@ include::{sourcedir36}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[t
 <5> Define the concurrency level of write-behind queue(s).
 This indicates how many writer threads work in parallel to update the underlying system of record asynchronously.
 <6> Enable the write coalescing behavior, which ensures that only one update per key per batch reaches the underlying system of record.
+
+== Cache-Through for Clustered Cache
+
+A clustered cache can also be configured with a `CacheLoaderWriter` for a cache-through use case.
```

Comment:
Is there anything special about clustered caches and write-through/write-behind beyond the fact that they don't honor the batching configuration (right now)? If not do we need a whole new section with samples here?

Reply:
No there is nothing different other the fact that Batched configs are not yet used in clustered caches.
IMO we don't need another sample or any other extra documentation